classes/core/PKPContainer.inc.php

```diff
@@ -207,7 +208,11 @@ protected function loadConfiguration()
             'channel' => 'errorlog',
         ];
 
-        $items['mail']['default'] = static::getDefaultMailer();
+        $items['mail']['default'] = static::getDefaultMailer() === 'disabled' ? 'sendmail' : static::getDefaultMailer();
```

Comment:
What do you think in move this logic to the `getDefaultMailer` method instead of using a ternary here?

Reply:
I did it, but for some reason, I moved back, I'll check later, perhaps this part will disappear in a next iteration.